src/bucket.js

```diff
@@ -71,6 +81,7 @@ export default class Bucket {
    * @param  {Object} options.headers The headers object option.
    * @return {Promise<Object, Error>}
    */
+  @nobatch("Batch doesn't support read operations.")
```

Comment:
Because the result of a read operation within a batch would be sent along all the responses received from the main batch one, and you usually want to consume directly what you're reading.